matrix-sdk-android/src/main/java/org/matrix/android/sdk/internal/session/sync/SyncTokenStore.kt

```diff
@@ -25,13 +25,18 @@ import javax.inject.Inject
 internal class SyncTokenStore @Inject constructor(@SessionDatabase private val monarchy: Monarchy) {
 
     fun getLastToken(): String? {
-        return Realm.getInstance(monarchy.realmConfiguration).use {
+        val token = Realm.getInstance(monarchy.realmConfiguration).use {
+            // Makes sure realm is up-to-date as it's used for querying internally on non looper thread.
+            it.refresh()
```

Comment:
Will it fix the notification about init sync not displayed after a clear cache? (not sure there is an existing issue about that)